src/test/java/elizCurtnazar/tests/tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,141 @@
+package elizCurtnazar.tests.tema6;
+
+import elizCurtnazar.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.ElementClickInterceptedException;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import teofilursan.driver.BrowserManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToDemoQAWindowPage();
+//        clickOnNewTabButton(); EXEMPLU DIN ORE
+        clickNewTab();
+        clickNewWindow();
+//        clickNewWindowMessage();
+        try {
+            clickNewWindowMessage();
+            System.out.println("Am intrat in blocul de try.");
+        } catch (Exception | Error e) {
+            System.out.println("Am prins o exceptie in blocul de catch.");
+            FileUtils.takeScreenshot(driver, "New Window Message");
+            //alta varianta
+//            File file = driver.getScreenshotAs(OutputType.FILE);
+//            File destinationFile = new File("C:\\Users\\Deniz\\IdeaProjects\\poza3.png");
+//            FileUtils.copyFile(file, destinationFile);
+        } finally {
+            driver.close();
+            System.out.println("Am inchis pagina de pe blocu de finally.");
+        }
+        closeBrowser();
```

Comment:
Replace driver.close from finally block to closeBrowser() and remove this closeBrowser from this line. Thanks!

Reply:
done